spec/fixtures/module/preload-pdf-loaded-in-subframe.js

```diff
@@ -1,7 +1,7 @@
 const { ipcRenderer } = require('electron')
 
 document.addEventListener('DOMContentLoaded', (event) => {
-  var subframe = document.getElementById('pdf-frame')
+  var subframe = document.querySelector('#pdf-frame')
```

Comment:
@MarshallOfSound 
What's a reason for this change?

Reply:
The linter declared it against the rules :)